site/docs/master/faq.md

```diff
@@ -0,0 +1,216 @@
+    operator: Exists
+sonobuoy-config:
+  driver: Job
+  plugin-name: my-custom-plugin
+  result-format: raw
+spec:
+  command:
+  - ./run.sh
+  image: myregistry/pluginimage:latest
+  name: plugin
+  resources: {}
+  volumeMounts:
+  - mountPath: /tmp/results
+    name: results
+```
+
+You can adapt the plugin definition to your requirements, add any tolerations that are required, then run that plugin as follows:
+
+```
+sonobuoy run -p <path/to/my-plugin.yaml>
+```
+
+If you wish to modify the PodSpec for the built in plugins (`e2e` and `systemd-logs`), you will first need generate a Sonobuoy manifest which includes those plugin definitions with the default PodSpec included:
+
+```
+sonobuoy gen --show-default-podspec
+```
+
+Within the output, you can find the `e2e` and `systemd-logs` plugin definitions in the `sonobuoy-plugins-cm` ConfigMap.
+You can then modify the tolerations in the PodSpec for each plugin and then run the manifest using: 
+
+```
+sonobuoy run -f <path/to/my-manifest.yaml>
+```
+
+For more details, please refer to our [documentation][customPlugins] or see our [blog post about this topic][custom-podspecs].
+
+### What tests can I run? How can I figure out what tests/tags I can select?
+The `e2e` plugin has a number of preconfigured modes for running tests, with the default mode running all conformance tests which are non-disruptive.
+It is possible to [configure the plugin][e2ePlugin] to provide a specific set of E2E tests to run instead.
+
+Which tests you can run depends on the version of Kubernetes you are testing as the list of tests changes with each release.
+
+A list of the conformance tests is maintained in the [Kubernetes repository][kubernetes-conformance].
+Within the GitHub UI, you can change the branch to the tag that matches your Kubernetes version to see all the tests for that version.
+This list provides each test name as well where you can find the test in the repository.
+You can include these test names in the `E2E_FOCUS` or `E2E_SKIP` environment variables when [running the plugin][e2ePlugin].
+
+Although the default behaviour is to run the Conformance tests, you can run any of the other Kubernetes E2E tests with Sonobuoy.
+These are not required for checking that your cluster is conformant and we only recommend running these if there is specific behaviour you wish to check.
+
+There are a large number of E2E tests available (over 4000 as of v1.16.0).
+Many of these tests have "tags" which show that they belong to a specific group, or have a particular trait.
+There isn't a definitive list of these tags, however below are some of the most commonly seen tags:
+
+- Conformance
+- NodeConformance
+- Slow
+- Serial
+- Disruptive
+- Flaky
+- LinuxOnly
+- Feature:* (there are numerous feature tags)
+
+There are also specific tags for tests that belong to a particular [Special Interest Group (SIG)][sig-list].
+As of writing this document, we were able to see the following SIG tags within the E2E tests:
```

Comment:
I'd just say "the tests included the following SIG tags:" instead of qualifying it with "we were able to see...within the E2E tests"